lib/src/main/java/com/bytehamster/lib/preferencesearch/SearchPreferenceActionView.java

```diff
@@ -55,6 +56,11 @@ public SearchConfiguration getSearchConfiguration() {
         return searchConfiguration;
     }
 
+    public void setSearchConfiguration(@NonNull SearchConfiguration searchConfiguration) {
```

Comment:
Hmm, I would prefer to not add this, in order to keep the interface more clean. With the visibility changes, you should be able to do this in a sub-class if you really need it.

Reply:
Oh, yeah. Now that searchConfiguration is protected, this isn't necessary. Good thought

Reply:
Dropped it